pkg/helm/helm.go

```diff
@@ -112,27 +113,58 @@ func InstallOrUpgradeHelmChartFromPlugin(ctx context.Context, local client.Clien
 	return nil
 }
 
-// ChartTest to do helm test on the plugin
-func ChartTest(restClientGetter genericclioptions.RESTClientGetter, plugin *greenhousev1alpha1.Plugin) (bool, error) {
-	var hasTestHook bool
+// ChartTest executes Helm chart tests and logs test pod logs if a test fails.
+func ChartTest(ctx context.Context, restClientGetter genericclioptions.RESTClientGetter, plugin *greenhousev1alpha1.Plugin) (hasTestHook bool, testPodLogs string, err error) {
 	cfg, err := newHelmAction(restClientGetter, plugin.Spec.ReleaseNamespace)
 	if err != nil {
-		return hasTestHook, err
+		return hasTestHook, "", err
 	}
-	rta := action.NewReleaseTesting(cfg)
-	rta.Timeout = GetHelmTimeout() // set a timeout for the release testing to avoid waiting forever
-	results, err := rta.Run(plugin.Name)
+
+	testAction := action.NewReleaseTesting(cfg)
+	testAction.Timeout = GetHelmTimeout() // set a timeout for the release testing to avoid waiting forever
+	// Used for fetching logs from test pods
+	testAction.Namespace = plugin.Spec.ReleaseNamespace
+	results, err := testAction.Run(plugin.Name)
 	if err != nil {
-		return hasTestHook, err
+		release, getErr := getLatestRelease(cfg, plugin.Name)
```

Comment:
In this case I think it is fine to just call the action directly, without wrapping it in a function.
```suggestion
		release, getErr := action.NewGet(cfg).Run(releaseName)
```

Reply:
sure. I did in 750e430bf6dcf7ef0628367e1965706330770539 :)